queries.sql

```diff
@@ -1,43 +1,75 @@
--- What animals belong to Melody Pond?
-SELECT animals.name
-FROM animals
-INNER JOIN owners ON animals.owner_id = owners.id
-WHERE owners.full_name = 'Melody Pond';
-
--- List of all animals that are Pokemon
-SELECT animals.name
-FROM animals
-INNER JOIN species ON animals.species_id = species.id
-WHERE species.name = 'Pokemon';
-
--- List all owners and their animals (including those who don't own any animal)
-SELECT owners.full_name, animals.name
-FROM owners
-LEFT JOIN animals ON owners.id = animals.owner_id;
-
--- How many animals are there per species?
-SELECT species.name, COUNT(*) AS animal_count
-FROM species
-LEFT JOIN animals ON species.id = animals.species_id
-GROUP BY species.name;
-
--- List all Digimon owned by Jennifer Orwell
-SELECT animals.name
-FROM animals
-INNER JOIN owners ON animals.owner_id = owners.id
-INNER JOIN species ON animals.species_id = species.id
-WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
-
--- List all animals owned by Dean Winchester that haven't tried to escape
-SELECT animals.name
-FROM animals
-INNER JOIN owners ON animals.owner_id = owners.id
-WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
-
--- Who owns the most animals?
-SELECT owners.full_name, COUNT(*) AS animal_count
-FROM owners
-LEFT JOIN animals ON owners.id = animals.owner_id
-GROUP BY owners.full_name
-ORDER BY animal_count DESC
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name AS last_animal_seen
+FROM visits v
+JOIN vets w ON v.vet_id = w.id
+JOIN animals a ON v.animal_id = a.id
+WHERE w.name = 'Vet William Tatcher'
+ORDER BY v.visit_date DESC
 LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT v.animal_id) AS number_of_animals_seen
+FROM visits v
+JOIN vets s ON v.vet_id = s.id
+WHERE s.name = 'Vet Stephanie Mendez';
```

Comment:
👍 Good progress on your project! However, there's a minor adjustment needed for your query to align with the project requirements:

- [ ] As per the project requirement, the query should aim to find out "How many different animals did Stephanie Mendez see?" The query you've mentioned seems to include the prefix "Vet" before Stephanie Mendez's name. Please update your query to match the required format without the "Vet" prefix and the screenshot for the updated query.

By making this small correction, your query will precisely fulfill the project's specifications. Keep up the good work! 👍🔍🦁